roles/ansible_service_broker/tasks/generate_certs.yml

```diff
@@ -0,0 +1,33 @@
+---
+
+- name: Create service catalog cert directory
+  file:
+    path: "{{ openshift.common.config_base }}/ansible-service-broker"
+    state: directory
+    mode: 0755
+  check_mode: no
+
+- set_fact:
+    generated_certs_dir: "{{ openshift.common.config_base }}/ansible-service-broker"
+
+- name: Create self signing ca cert
+  command: 'openssl req -nodes -x509 -newkey rsa:4096 -keyout {{ generated_certs_dir }}/key.pem -out {{ generated_certs_dir }}/cert.pem -days 365 -subj "/CN=asb-etcd.ansible-service-broker.svc"'
+  args:
+    creates: '{{ generated_certs_dir }}/cert.pem'
+
+- name: Create self signed client cert
+  command: '{{ item.cmd }}'
+  args:
+    creates: '{{ item.creates }}'
+  with_items:
+    - cmd: openssl genrsa -out {{ generated_certs_dir }}/client.key 2048
+      creates: '{{ generated_certs_dir }}/client.key'
+    - cmd: 'openssl req -new -key {{ generated_certs_dir }}/client.key -out {{ generated_certs_dir }}/client.csr -subj "/CN=client"'
+      creates: '{{ generated_certs_dir }}/client.csr'
+    - cmd: openssl x509 -req -in {{ generated_certs_dir }}/client.csr -CA {{ generated_certs_dir }}/cert.pem -CAkey {{ generated_certs_dir }}/key.pem -CAcreateserial -out {{ generated_certs_dir }}/client.pem -days 1024
+      creates: '{{ generated_certs_dir }}/client.pem'
+
+- set_fact:
+    etcd_ca_cert: "{{ lookup('file', '{{ generated_certs_dir }}/cert.pem') }}"
```

Comment:
It would be nice if a user could override these and we would use the certs they pass in?

Reply:
@ewolinetz any objection to this or preferred way to pass in the certs within the openshift-ansible installer?